token/core/identity/msp/idemix/lm.go

```diff
@@ -11,6 +11,8 @@ import (
 	"path/filepath"
 	"sync"
 
+	config2 "github.com/hyperledger-labs/fabric-token-sdk/token/core/config"
+
```

Comment:
could you please use the chance to get rid of this empty line? Our imports look like minced meat at many places :D

Reply:
done